Add section 2.6 (2.5) (#14)
jneidel/internet.org-entwicklung: main.tex

In der Zwischenzeit versuchte Facebook, nach initellen Startschwierigkeiten, die Initiative durch Rebranding zu retten, was aber nicht reichte um die kritische Öffentlichkeit zu beeinflußen, deren Meinung bis zu den Entscheidungträgern der zuständigen Behörde vordrigen konnte.
Welche sich dann mit einem indirekten Verbot auf der Seite der Protestierenden positionierte.

% 2.5 Nationaler Start (Google leaves, public opinion, TRAI accusations, FB urges user to work for them)
\subsection{Nationaler Start}

% 2.6 - Verbot durch Regulatoren (TRAIs "Prohibitions of Discriminiatory tariffs for Data Services Regulation 2016", aftermath)
\subsection{Verbot durch Regulatoren}

%TRAI in den Index?

%Politics of Free Basics and its Death in India p7

%Internet.org suspended durch TRAI in 2015
%TRAI schuldigt Facebook der Täuschung der

> Noch in Mitten des Austausches über das Pro und Contra der Netzneutralität veranlasste TRAI im Dezember 2015 eine Anordnung an Reliance Communications Facebook's Free Basics vorläufig zu sperren.

Author: ```suggestion
Noch in Mitten des Austausches über das Pro und Contra der Netzneutralität veranlasste TRAI (Telecom Regulatory Authority of India) im Dezember 2015 eine Anordnung an Reliance Communications, Facebook's Free Basics vorläufig zu sperren.
```

Damit man weiß wofür TRAI steht

Reviewer: achja, da war ich mir noch nicht sicher, ob das schon vorher erwähnt wird oder erst hier

Reviewer: Müsste aber mindestens schon in 2.5 erwähnt werden

\autocite{Internet governance as 'ideology in practice' - India's 'Free Basics' controversy}

> Im Februar 2016 verabschiedete TRAI die "Prohibition of Discriminatory Tariffs for Data Services Regulations".

Author: ```suggestion
Im Februar 2016 verabschiedete TRAI daraufhin die `Prohibition of Discriminatory Tariffs for Data Services Regulations'.
```

> In dieser Verordnung legte die Behörde fest, dass von Telekommunikationsanbietern geschlossene Verträge und Tarife mit Kunden nicht zwischen Inhalten und Websites differenzieren dürfen. Sie beruft sich dabei auf die Prinzipien der Netzneutralität, welche verlangt, dass das Medium zwischen den Endnutzern neutral bleibt.

Author: ```suggestion
In dieser Verordnung legte die Behörde fest, dass von Telekommunikationsanbietern geschlossene Verträge und Tarife mit Kunden nicht zwischen Inhalten bzw. Websiten differenzieren dürfen. Sie beruft sich dabei auf die Prinzipien der Netzneutralität, welche verlangt, dass das Medium zwischen den Endnutzern neutral bleibt.
```

Sonst klingt es als ob nicht zwischen Inhalt, auf der einen Seite, und Websiten auf der anderen, unterschieden werden darf.

Author: `beruft sich dabei auf die Prinzipien der Netzneutralität, welche verlangt`

Entwerden es sind die Prinzipien, welche etwas `verlangen`
Oder es ist das `das Prinzip`

Author: `dass das Medium zwischen den Endnutzern neutral bleibt`

Klingt krude :)

%"darauf" im folgenden Satz ersetzen

> Basierend darauf wurde entschieden, dass Free Basics, welches gratis Zugang zu kooperierenden Diensten zuließ, den

Author: ```suggestion
Basierend darauf wurde entschieden, dass Free Basics, welches freien Zugang zu kooperierenden Diensten ermöglicht, den
```

> Verordnungen widerspricht und wurde am 8. Februar 2016 Internet.org/Free Basics offiziell in Indien gebannt wurde.

Author: ```suggestion
Verordnungen widerspricht und folgegemäß ab dem 8. Februar 2016 offiziell in Indien verboten sei.
```

> Dies führte dazu, dass Facebook am 11 Februar 2016 die Plattform Free Basics zurückzog.

Author: ```suggestion
```

Wird eigentlich nicht gebraucht :smile:

In der Zwischenzeit versuchte Facebook, nach initellen Startschwierigkeiten, die Initiative durch Rebranding zu retten, was aber nicht reichte um die kritische Öffentlichkeit zu beeinflußen, deren Meinung bis zu den Entscheidungträgern der zuständigen Behörde vordrigen konnte.
Welche sich dann mit einem indirekten Verbot auf der Seite der Protestierenden positionierte.

% 2.5 Nationaler Start (Google leaves, public opinion, TRAI accusations, FB urges user to work for them)

> \subsection{Nationaler Start}

Reviewer 2: Label: {national}

%23.11.2015 \autocite{facebook launch post}
Der nationale Start von Internet.org's Freebasic App begann am 23. November 2015 und wurde noch am selben Tag von Mark Zuckerberg unter anderem über Facebook selbst bekanntgegeben. Nutzer hatten Zugriff auf Wikipedia, Bing, BBC News, soweiso einige lokale Nachrichtendienste.
	
Google hingegen war kein Teil der Freebasics Plattform, obwohl sie Teil der Testphase in Sambia waren. Dies führte zu mehr Skepsis gegenüber Facebooks Plattform. Hinzu kommt, dass sich Google CEO Sundar Pichai positiv in der Debatte um Netzneutralität ausgesprochen hat.
%cite: https://tech.economictimes.indiatimes.com/news/internet/ease-of-doing-business-has-improved-in-india-but-can-get-a-whole-lot-better-sundar-pichai/50212415
%\subsubsection{public opinion}
	
Ein größeres Problem bestand auch darin, wie Facebooks Projekt in Indien wahrgenommen wurde im Gegensatz zu ihrer eigenen Werbekampagne.

> Freebasics, sofern überhaupt von der Bevölkerung wahrgenommen, wird von vielen Indern als Werbung für Facebook und Reliance Verträge gesehen wird. Dies steht kritisch in Relation zu ihren eigenen Behauptungen, man versuche der armen Landbevölkerung gleichwertigen Zugriff auf Informationen durch das Medium Internet zu gewähren.

Reviewer 2: Freebasics [...] wird von vielen Indern als Werbung für Facebook und Reliance angesehen.
